Source/Tools/LinuxEmulation/LinuxSyscalls/FileManagement.cpp

```diff
@@ -503,6 +503,21 @@ std::pair<int, const char*> FileManager::GetEmulatedFDPath(int dirfd, const char
   return std::make_pair(RootFSFD, &SubPath[1]);
 }
 
+bool FileManager::IsSelf(const char* Pathname) const {
+  if (!Pathname) {
```

Comment:
Can Pathname actually be null..?

Reply:
Yes, it is allowed for some syscalls (like statx) to pass a nullptr pathname, which then it uses the dirfd passed in.

Reply:
Fun, thanks